modules/app/app-system/src/main/resources/tasks/auditLogCleanUp/auditLogCleanUp.xml

```diff
@@ -0,0 +1,5 @@
+<task>
+  <description>AuditLog clean up task</description>
+  <form>
```

Comment:
it should be possible to specify threshold here (but job should not specify anything)